File: lesson4/python3/homework/homework4_10.py
#!/usr/bin/python3

numb = int(input("Введите натуральное число => "))

numb_i = 1
while numb_i <= numb:
    number_result = numb // numb_i
    remainder_result = numb % numb_i
    if (number_result <= numb and remainder_result == 0) or number_result == 1:
        if number_result == 1:
            print("number_result =>", number_result)
            break;
        print("number_result =>", number_result)
    numb_i += 1

File: lesson4/c_c++/homework/homework4_10.cpp
//: c_c++/homework4/homework4_10.cpp
#include <iostream>

int main() {
    using namespace std;

    int numb;
    cout << "Введите натуральное число => ";
    cin >> numb;

    int numb_i;
    while (numb_i < numb) {
        int result = numb / numb_i;
        int remainder_result = numb % numb_i;
        if ((result <= numb && remainder_result == 0) || result == 1) {
            if (result == 1) {
                cout << "number_result => " << result << endl;
                break;
            }
            cout << "number_result => " << result << endl;
        }
        numb_i++;
    }
    return 0;
}
